clients/drcachesim/tests/cache_replacement_policy_unit_test.cpp

```diff
@@ -189,20 +155,9 @@ unit_test_cache_fifo_four_way()
     const addr_t ADDRESS_G = 448;
     const addr_t ADDRESS_H = 512;
 
-    assert(cache_fifo_test.get_block_index(ADDRESS_A) ==
-           cache_fifo_test.get_block_index(ADDRESS_B));
-    assert(cache_fifo_test.get_block_index(ADDRESS_B) ==
-           cache_fifo_test.get_block_index(ADDRESS_C));
-    assert(cache_fifo_test.get_block_index(ADDRESS_C) ==
-           cache_fifo_test.get_block_index(ADDRESS_D));
-    assert(cache_fifo_test.get_block_index(ADDRESS_D) ==
-           cache_fifo_test.get_block_index(ADDRESS_E));
-    assert(cache_fifo_test.get_block_index(ADDRESS_E) ==
-           cache_fifo_test.get_block_index(ADDRESS_F));
-    assert(cache_fifo_test.get_block_index(ADDRESS_F) ==
-           cache_fifo_test.get_block_index(ADDRESS_G));
-    assert(cache_fifo_test.get_block_index(ADDRESS_G) ==
-           cache_fifo_test.get_block_index(ADDRESS_H));
+    assert(cache_fifo_test.block_indices_are_identical(
+        std::vector<int> { ADDRESS_A, ADDRESS_B, ADDRESS_C, ADDRESS_D, ADDRESS_E,
+                           ADDRESS_F, ADDRESS_G, ADDRESS_H }));
```

Comment:
I think this is a lot nicer: compact, easier to read, less error-prone.  The remaining thing that jumps out is having to repeat the names and make the vector twice.  Maybe the constant addresses should be a vector to begin with, the ordinals as the named constants for the test lines below?  Or at least create the vector just once and share it.

Ditto for the LRU test.

Reply:
Done.